oslodb_sqlalchemy_exceptions/__init__.py,sha256=IMjkMO3twhQzluVTo8Z6rE7Eg-9U79_LGKMcsWLKBkY,22
oslodb_sqlalchemy_exceptions/exceptions.py,sha256=cu7HxA-mdhKJI1ybkJRAI0AO4FYxueLGQlP8qtcIscY,10622
oslodb_sqlalchemy_exceptions/reflection.py,sha256=Bu3Isa854k1LygJ9TmLj8WxGL91e7_e6mjWcvmRMgXc,8216
oslodb_sqlalchemy_exceptions/sqla_errors.py,sha256=rMSwOe8IJ7GYisnORkTPxoBV_8YwfucgcnkGIVvO78o,19887
oslodb_sqlalchemy_exceptions/warning.py,sha256=C_8FyC5nY17c9YHvj5QrOyZJqUit3L9GLNtBlHJ_Whc,1148
oslodb_sqlalchemy_exceptions-0.1.0.dist-info/WHEEL,sha256=V7iVckP-GYreevsTDnv1eAinQt_aArwnAxmnP0gygBY,83
oslodb_sqlalchemy_exceptions-0.1.0.dist-info/METADATA,sha256=RXR6i6IKl73zepDAx0AddLg3t9zJqC5_928r8KY6c_o,437
oslodb_sqlalchemy_exceptions-0.1.0.dist-info/RECORD,,
